src/buffer.jl

```diff
@@ -200,11 +191,6 @@ function copydata!(buf::Buffer, data::Ptr{UInt8}, nbytes::Integer)
     return buf
 end
 
-# Copy data from `data` to `buf`.
-function copydata!(buf::Buffer, data::Buffer, nbytes::Integer = length(data))
```

Comment:
This seems to actually be a safer API than the one below.

Reply:
Yes, I guess it could be with some added bounds checks, but these functions are mostly used in `unsafe_read` and `unsafe_write` where things are already using `Ptr{UInt8}`.